linkerd/config/src/main/scala/io/buoyant/linkerd/config/Parser.scala

```diff
@@ -11,7 +11,9 @@ import com.twitter.finagle.util.LoadService
 import scala.reflect.{ClassTag, classTag}
 
 abstract class ConfigDeserializer[T: ClassTag] extends StdDeserializer[T](Parser.jClass[T]) {
-  def register(module: SimpleModule): SimpleModule = module.addDeserializer(Parser.jClass[T], this)
+  def register(module: SimpleModule): SimpleModule = {
```

Comment:
Yeah, I think this was a remnant of a refactor where I had to add a few more lines. Braces removed.